dbt/include/athena/macros/materializations/models/table/table.sql

```diff
@@ -65,18 +65,18 @@
       {%- if old_relation is not none -%}
         {{ drop_relation(old_relation) }}
       {%- endif -%}
-      {%- set query_result = safe_create_table_as(False, target_relation, sql) -%}
+      {%- set query_result = safe_create_table_as(False, target_relation, sql, False) -%}
```

Comment:
why do you pass `False` as batch_insert? should actually come from config.

Reply:
Yep, fixed.
Sorry, my bad.